crates/walrus-service/src/client/cli/args.rs

```diff
@@ -50,6 +50,7 @@ pub struct App {
     /// 2. If the environment variable `XDG_CONFIG_HOME` is set, in `$XDG_CONFIG_HOME/walrus/`.
     /// 3. In `~/.config/walrus/`.
     /// 4. In `~/.walrus/`.
+    /// 5. The environment variable `WALRUS_CONFIG` as complete path
```

Comment:
Suggestion: I think this should have higher priority than the default paths: It should be checked immediately after the explicitly specified path. (In particular as it's supposed to be the full path (including the filename) AFAIU as opposed to the default directories.)